HealthVault/Tools/codegen/codegen.sh

```diff
@@ -76,3 +76,9 @@ s/"MHVTrackingValidation.h"/"MHVTrackingValidation.h"\
 #import "MHVPlanStatusEnum.h"/g
 ' output/SwaggerClient/Api/MHVActionPlanTasksApi.m > output/SwaggerClient/Api/MHVActionPlanTasksApi.m2
 mv output/SwaggerClient/Api/MHVActionPlanTasksApi.m2 output/SwaggerClient/Api/MHVActionPlanTasksApi.m
+
+# Other nullability fixes
+sed -E '
+s/strong,nonatomic) NSString\* associatedPlanId/strong,nonatomic,nullable) NSString* associatedPlanId/g
+' output/SwaggerClient/Model/MHVActionPlanTask.h > output/SwaggerClient/Model/MHVActionPlanTask.h2
```

Comment:
I filed a bug to fix to fix the DTO so Swagger generates it with the nullable attribute, this can be removed when that is fixed

Reply:
We should just get the bug fixed and not add this one off code here. You can raise the priority of the bug if this is blocking you in some way, but this is not a good idea.